### PR TITLE
Convert from Cloud Run to AppEngine

### PR DESCRIPTION
**Pull request by LogicalShark**
@tbpg

### CODE REVIEW

File: getting-started/gopher-run/app.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+runtime: go112
+main: cmd
+env_variables: 
+  GOPHER_RUN_BUCKET: "maralder-start-ml"
+  GOOGLE_CLOUD_PROJECT: "maralder-start"
```
Comment (tbpg) [inline getting-started/gopher-run/app.yaml]: Update for the new project.

File: getting-started/gopher-run/app.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+handlers:
+  - url: /play
+    static_dir: static/gorun
```
Comment (tbpg) [inline getting-started/gopher-run/app.yaml]: Newline at the end.

File: getting-started/gopher-run/app.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+runtime: go112
```
Comment (tbpg) [inline getting-started/gopher-run/app.yaml]: License at the top.

Comment (LogicalShark) [reply getting-started/gopher-run/app.yaml]: Done

File: getting-started/gopher-run/app.yaml (hunk 1 above)
Comment (LogicalShark) [reply getting-started/gopher-run/app.yaml]: Done

File: getting-started/gopher-run/app.yaml (hunk 2 above)
Comment (LogicalShark) [reply getting-started/gopher-run/app.yaml]: Done

**Pull request merged**